services/QuillLMS/engines/evidence/app/models/evidence/research/gen_ai/llm_prompt.rb

```diff
@@ -75,7 +75,7 @@ def prompt_with_student_response(student_response)
 
         def guidelines_count = optimal_guidelines_count + suboptimal_guidelines_count
 
-        def to_s = description.truncate(40)
+        def to_s = name.truncate(40)
```

Comment:
Just a note, this is probably fine for now, but might be something that causes issues if long names prove helpful for distinguishing between subtly different prompts (but this truncate cuts off the part that distinguishes it, e.g. `"- V2"`. Might be better to wrap this in the UI rather than cut it off.

Reply:
Good point.  I will remove.